controls/cf_serverd.cf

```diff
@@ -144,7 +144,7 @@ bundle server mpf_default_access_rules()
       "$(sys.workdir)/cmdb/$(connection.key)/" -> { "ENT-6788" }
         handle => "server_access_grant_access_cmdb",
         comment => "Grant access to host specific CMDB data",
-        if => isdir( "$(sys.workdir)/cmdb/$(connection.key)" ),
+        shortcut => "hub_mdr",
```

Comment:
What kind of black magic is this?! Interesting... Anyway, I'd stick to *cmdb*, *mdr* is just confusing here.

Reply:
Woah, the shortcut expands individual `$(connection.key)` variables? Cool! 🚀

Reply:
Yeah, shortcuts are cool stuff. It's how we can let the remote host decide what file we get.